apps/docs/src/guide/wallets/wallet-transferring.md

```diff
@@ -18,6 +18,12 @@ When transferring the base chain coin like ETH, you can omit the `assetId`:
 
 <<< @/../../docs-snippets/src/guide/wallets/wallet-transferring.test.ts#wallet-transferring-3{ts:line-numbers}
 
+## Transferring To Multiple Wallets
+
+To transfer assets to multiple wallets, use the `Account.multiTransfer` method:
```

Comment:
We may want to call the method `batchTransfer` as that's the more familiar [parlance in the space](https://coinguides.org/batching-bitcoin-transactions/)

Reply:
@maschad That is a good suggestion, I agree that `batchTransfer` makes it clear that the method is designed to handle multiple transfers in a single transaction

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2333/commits/b06df0153361f0de37c1d151ba0767a17f9fb9af